src/content/docs/de/tutorial/3-components/index.mdx

```diff
@@ -0,0 +1,39 @@
+---
+type: tutorial
+unitTitle: Erstelle und gestalte mit Astro-Komponenten
+title: 'Zwischenstand: Einheit 3 - Komponenten'
```

Comment:
Findet ihr es besser, wenn am Anfang jeder Einheit `Check in` oder `Zwischenstand` im Titel steht?
Bin mir nicht sicher, was natürlicher wirkt.

Reply:
ist wieder die Frage was wir alles übersetzen wollen, ich denke hier ist eine Übersetzung in Ordnung, habe aber auch nichts gegen check in, sollte im Deutschen allgemein bekannt sein.

Reply:
Stimmt, das glaube ich auch. `Zwischenstand` finde ich als Übersetzung auch irgendwie subtoptimal, weil es die Bedeutung des Begriffs verzerrt.